code-challenges.test.js

```diff
@@ -15,42 +15,173 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// --- > ::STUDENT NOTES:: This question confused me at first because this problem was not supplied with variables like it states in the test question/statement, only suggested example inputs? With that said, I bypassed this statement and proceeded to make the test work as I suspect I am supposed to.
```

Comment:
Yes, you approached this the right way.  Thank you for writing this in here!  If you every need clarification, please feel free to reach out to one of us when you are working on your assessment, we are here to help!